README.md

```diff
@@ -1,2 +1,17 @@
-# thermal_base
-Decoding and common processing for thermographs / thermograms. Currently supports FLIR's TIFF and PNG encoded metadata, and DJI-encoded metadata
+# Base codes for Thermography
+
+A python package for decoding and common processing for thermographs / thermograms. Currently supports FLIR's TIFF and PNG encoded metadata, and DJI-encoded metadata
+
+## Install
+Install this package with `pip install git+ssh://git@github.com/detecttechnologies/thermal_base.git@main`
+
+**Disclaimer**
+
+Tested against data from: 
+* Flir encoding format: DJI XT-2, Flir E-40(Temperature values go out of bounds of the image for <255 height)
```

Comment:
Can change the last part to
` (Temperature values go out of the image frame if the image height is <255 pixels)`